### PR TITLE
Fixes #53, claim food truck

### LINKED ISSUE #53: As a food truck owner, I want to be able to claim a food truck I own that was scraped

### PR DESCRIPTION
**Pull request by kftang**

### CODE REVIEW

File: routes/foodtrucks.go [hunk 1]
```diff
@@ -304,5 +304,52 @@ func PutFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
...
+	userID, userLoggedIn := r.Context().Value(middleware.UserKey).(string)
+
+	// Check for a user
+	if !userLoggedIn {
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	// Lookup food truck in db
+	var foodTruck models.JSONFoodTruck
```
Comment (janinebar) [inline routes/foodtrucks.go:344]: Just so i understand, but this is a check for if the food truck is in the db?

File: routes/foodtrucks_test.go [hunk 2]
```diff
@@ -55,7 +57,8 @@ func TestFoodTrucksGetSingle(t *testing.T) {
 
 func TestFoodTruckGetInvalid(t *testing.T) {
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: Does endpoint match test goals (looking at func name)?

File: routes/foodtrucks_test.go [hunk 3]
```diff
@@ -55,7 +57,8 @@ func TestFoodTrucksGetSingle(t *testing.T) {
...
 	tests.ClearDB()
-	req, _ := http.NewRequest("GET", "/foodtruck", nil)
+
+	req, _ := http.NewRequest("GET", "/foodtrucks", nil)
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: Here as well ^^^

File: routes/foodtrucks_test.go [hunk 4]
```diff
@@ -72,11 +75,12 @@ func TestFoodTruckGetInvalid(t *testing.T) {
 
 func TestFoodTruckGetValid(t *testing.T) {
 	tests.ClearDB()
+
 	addFoodTruck := models.JSONFoodTruck{
 		ID: "test",
 	}
 	tests.AddFoodTruck(addFoodTruck)
-	req, _ := http.NewRequest("GET", "/foodtruck", nil)
+	req, _ := http.NewRequest("GET", "/foodtrucks", nil)
```
Comment (janinebar) [inline routes/foodtrucks_test.go]: Refer to above comment

**Review by janinebar — CHANGES_REQUESTED** [review]

File: routes/foodtrucks.go (hunk 1 above)
Comment (kftang) [reply routes/foodtrucks.go:344]: yeah, just that

File: routes/foodtrucks_test.go (hunk 3 above)
Comment (kftang) [reply routes/foodtrucks_test.go]: yeah this is wrong

Comment (kftang) [reply routes/foodtrucks_test.go]: it's ok because i still set the handler to the correct one (GetFoodTruckHandler) singular

**Review by janinebar — APPROVED** [review]

**Pull request merged**